### PR TITLE
Added new and improved existing rules for Stack Exchange websites

### LINKED ISSUE #41: Unhandled banners on stackexchange sites
There are some more StackExchange sites we can add rules for:

- [ ] superuser.com
- [ ] stackoverflow.co

### LINKED ISSUE #200: Existing cookie rejection rules on Stack Exchange websites do not work anymore
Even though Stack Exchange websites are theoretically supported and already have their own rules that should handle cookie banners when visiting them, the one responsible for rejecting cookies does not work correctly.

Upon closer examination, the cookie that Firefox injects, which should prevent the cookie banner from appearing, stores a very old date as its value (2023-01-04), which for some reason is then checked somewhere else to see if it is not older than 3 months from today. However, it doesn't seem to care if the date is too new, so a date such as 2030-01-01 is considered valid.

Rejected cookie warning in console:
Rejected-cookie-warning-in-console

### PR DESCRIPTION
**Pull request by Greg-21**
The existing rule responsible for rejecting cookies on the various Stack Exchange websites was not working correctly, so I updated the injected cookie and the date stored in it to one that won't become too old anytime soon. Apart from that, I added an opt-out click rule as a fallback in case this cookie stops working again somewhere in the future.

Also, I added a few other Stack Exchange websites that were missing.

I tested everything gradually, and it worked on my computer, but I'm not sure if it will work for other people as it did for me.

Resolves #41
Fixes #200

### CODE REVIEW

File: cookie-banner-rules-list.json [hunk 1]
```diff
@@ -292,25 +292,33 @@
...
+        "stackoverflowteams.com",
+        "superuser.com"
+      ],
       "cookies": {
         "optOut": [
           {
             "name": "OptanonAlertBoxClosed",
-            "value": "2023-01-04T15:56:30.908Z"
+            "value": "2030-01-01T00:00:00.000Z",
+            "sameSite": 1
```
Comment (Trikolon) [inline cookie-banner-rules-list.json:313]: I don't think passing an explicit value for sameSite is needed? In my tests it worked fine without it.

**Review by Trikolon — APPROVED** [review]
Thanks!!

File: cookie-banner-rules-list.json (hunk 1 above)
Comment (Trikolon) [reply cookie-banner-rules-list.json:313]: Thanks! Let's remove it so our code falls back to the default sameSite setting.

Comment (Greg-21) [reply cookie-banner-rules-list.json]: Done. However, SameSite=Lax will soon become the default unless Mozilla's plans change in this regard (or have already changed).

Comment (Greg-21) [reply cookie-banner-rules-list.json]: Done. However, SameSite=Lax will soon become the default value in Firefox unless Mozilla's plans change in this regard (or have already changed).

Comment (Greg-21) [reply cookie-banner-rules-list.json]: Done. It's worth noting that SameSite=Lax will soon become the default value in Firefox unless Mozilla's plans change in this regard (or have already changed).

**Pull request merged**